src/app/public/modules/tabs/tabset.component.scss

```diff
@@ -77,9 +80,15 @@
   color: $sky-text-color-action-primary;
 }
 
-@include sky-theme-modern {
+.sky-theme-modern {
   .sky-tabset {
     border-bottom: 1px solid $sky-theme-modern-border-color-neutral-medium;
     overflow: visible; /* for drop shadow visibility on focused tabs */
   }
+
+  .sky-tab-dropdown-menu-item {
+    .sky-btn-tab-wrapper {
+      margin-right: 0 !important;
```

Comment:
I think this can be done without the `!important` here.

Reply:
Yep, just had to remove encapsulation on the tab-button component.